modules/storage-persistent-storage-volume-encrypt-with-kms-key.adoc

```diff
@@ -66,6 +66,10 @@ kind: Pod
 metadata:
```

Comment:
This yaml file missing one line`apiVersion: v1`

Reply:
Thanks, added!